crates/biome_js_analyze/tests/specs/source/useSortedKeys/group-by-nesting-lexicographic.js.snap

```diff
@@ -0,0 +1,74 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+assertion_line: 152
+expression: group-by-nesting-lexicographic.js
+---
+# Input
+```js
+const obj = {
+  name: "Sample Item",
+  details: {
+    description: "This is a nested object",
+    status: "active"
+  },
+  id: "12345",
+  tags: ["short", "array"],
+  metadata: {
+    created: "2024-01-01",
+    updated: "2024-01-02"
+  },
+  count: 42,
+  multiLineArray: [
+    "item1",
+    "item2",
+    "item3"
+  ]
+};
+
+```
+
+# Diagnostics
+```
+group-by-nesting-lexicographic.js:2:3 assist/source/useSortedKeys  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i The object properties are not sorted by key.
```

Comment:
The diagnostic should be updated because it doesn't reflect the options of the assist